docs/src/systems/boundary.md

```diff
@@ -98,6 +98,12 @@ where the sum is over all fluid particles, ``\rho_f`` and ``p_f`` denote the den
 ```
 
 #### 2. [`BernoulliPressureExtrapolation`](@ref)
+Identical to [`AdamiPressureExtrapolation`](@ref) but adds the dynamic pressure part of the Bernoulli equation
+```math
+\frac{1}{2} \rho_f (v_f-v_{body})^2
```

Comment:
```suggestion
\frac{1}{2} \rho_f \Vert v_f-v_\text{body} \Vert^2
```

And can you please add `where ``v_f`` is the velocity of the fluid and ``v_\text{body}`` is the velocity of the body`?

Reply:
> Also mention that this is not found in the literature.

Reply:
> Also mention that this is not found in the literature.
If there is no reference this should be the assumption. Seems to me kinda silly to write "This modification is not referenced from literature.